app/controllers/profiles/accounts_controller.rb

```diff
@@ -6,8 +6,6 @@ module Profiles
   class AccountsController < Profiles::ApplicationController
     layout 'profiles'
```

Comment:
Since you are basing this off of `Profiles::ApplicationController` you can remove this layout line.

Reply:
Updated in 6dc4a9d5114f8c5f8ef460d3b5108c373769273e